packages/react-components/src/components/utils/__tests__/time-axis-component.test.tsx

```diff
@@ -1,15 +1,28 @@
 import * as React from 'react';
-import { cleanup } from '@testing-library/react';
-import { mount } from 'enzyme';
+import { cleanup, render, renderHook } from '@testing-library/react';
 import { TimeAxisComponent } from '../time-axis-component';
 import { TimeGraphUnitController } from 'timeline-chart/lib/time-graph-unit-controller';
 import { OutputComponentStyle } from '../output-component-style';
 
-afterEach(cleanup);
+// afterEach(cleanup);
```

Comment:
I'd rather remove this line versus leaving it commented out.

Reply:
Done